webapp/src/ts/services/select2-search.service.ts

```diff
@@ -182,6 +182,19 @@ export class Select2SearchService {
     }
   }
 
+  private setDoc(selectEl, doc) {
+    if (doc) {
+      selectEl.select2('data')[0].doc = doc;  // Set the value
+      // In case an unknown doc was set before, remove the grayed style from select2
+      selectEl.next('span').find('.select2-selection__rendered').removeClass('grayed');
```

Comment:
I'm a bit uncomfortable with using this selector in code. Is there a way that we can target `.select2-selection__rendered` directly from css and add the class to the select2 container? 
Also `grayed` isn't a very suggestive name. All dark text in webapp is grey (#333333). We could use something like "missing" or "unknown".

Reply:
> Is there a way that we can target .select2-selection__rendered directly from css and add the class to the select2 container?

I tried but existing rules make it hard, I'll try a bit more.

> We could use something like "missing" or "unknown"

Yes, agree, the CSS rule should have a more "semantic" name, so I'll rename it to "missing"